src/main/java/com/nttdata/nttdatanotificationservice/teams/models/TeamsCard.java

```diff
@@ -16,16 +16,26 @@
   private String context;
```

Comment:
private static final String TYPE = "MessageCard";
  private static final String CONTEXT = "http://schema.org/extensions";

Refactor to name these to something like DEFAULT_TYPE, DEFAULT_CONTEXT should fix the codeclimate